functions/go/apply-replacements/README.md

```diff
@@ -0,0 +1,107 @@
+# apply-replacements
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Invoke the [kustomize replacements] feature as a KRM function. 
+
+<!--mdtogo-->
+
+### FunctionConfig
+
+<!--mdtogo:Long-->
+
+We use a `Replacements` object to configure the `apply-replacements` function. 
+
+```yaml
+apiVersion: kpt.dev/v1alpha1
+kind: Replacements
```

Comment:
nit: how about `SetReplacements` or `ApplyReplacements`. curated functionConfig type has the naming convention like `SetNamespace` `SetAnnotations` `SetImage`

Reply:
+1
Maybe `ApplyReplacements`.

Reply:
done